src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/Addressable.java

```diff
@@ -40,8 +40,7 @@
 public sealed interface Addressable permits MemorySegment, MemoryAddress, NativeSymbol, VaList {
 
     /**
-     * Returns the memory address associated with this addressable.
-     * @return The memory address associated with this addressable.
+     * {@return the {@linkplain MemoryAddress memory address} associated with this addressable}
```

Comment:
Oh, i just learnt a new JavaDoc trick!

Reply:
heh - me too - hat tip to @pavelrappo :-)